packages/pubsub/src/index.js

```diff
@@ -553,8 +553,15 @@ PubSub.prototype.getTopicsStream = common.paginator.streamify('getTopics');
  *     messages. (default: 10)
  * @param {number} options.maxInProgress - Maximum messages to consume
  *     simultaneously.
+ * @param {number|date} options.messageRetentionDuration - Set this to override
+ *     the default duration of 7 days. This value is expected in seconds (length
+ *     of retention) or a Date (expiration of retention). Acceptable values are
```

Comment:
Thank you Alex for catching this. I don't think this makes sense.

The duration is from the moment each message is published, so different messages will have different "expiration dates". The doc seems to imply that all messages will expire at the same time which is not the case.